docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -298,6 +262,91 @@ At this point, if you switch to **Raw JSON** mode in your robot's **Config** tab
       },
       "depends_on": []
     },
+    {
+      "attributes": {
+        "board": "local",
+        "pins": {
+          "b": "lb",
+          "a": "la"
+        }
+      },
+      "depends_on": [],
+      "name": "l-encoder",
+      "type": "encoder",
+      "model": "incremental"
+    },
+    {
+      "model": "incremental",
+      "type": "encoder",
+      "namespace": "rdk",
+      "attributes": {
+        "board": "local",
+        "pins": {
+          "b": "rb",
+          "a": "ra"
+        }
+      },
+      "depends_on": [],
+      "name": "r-encoder"
+    }
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /expand %}}
+
+### Configure movement sensors
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+1.  Configure a GPS movement sensor so the robot knows where it is while navigating.
+    We configured ours as a `gps-nmea-rtk-serial` movement sensor:
+
+    ![An example configuration for a GPS movement sensor in the Viam app Config Builder.](/tutorials/navigate-with-rover-base/gps-movement-sensor-config-builder.png)
+
+    We named ours `gps`.
+    Refer to [the `gps-nmea-rtk-serial` movement sensor documentation](/components/movement-sensor/gps/gps-nmea-rtk-serial/) for attribute information.
+
```

Comment:
```suggestion

2. Configure a wheeled odometry movement sensor to provide angular and linear velocity measurements from the encoded motors on our base.

```

Reply:
This was wrong in the original tutorial, should have been separated out originally, but we can fix now